mineral-ui/mineral-ui
# [Select] Fix click to open in Edge

### Description

Fix `Select`'s click to open in Edge

### Motivation and context

It was reported that a `Select` component in a `FormField` did not open on a single click using Edge.  The issue can be replicated here.  https://mineral-ui.com/components/select/form-field

It turns out that the hidden input element inside of the `SelectTrigger` was bubbling click events, causing the `Select` to open and immediately close.  This was resolved by stopping the propagation of the hidden input's click event.

### Screenshots, videos, or demo, if appropriate

https://select-fix-edge--mineral-ui.netlify.com/components/select/form-field

### How to test

Browse to the demo example link above, using MS Edge, and verify that `Select` opens on a single mouse click.

### Types of changes

- Bug fix (non-breaking change which fixes an issue)

### Checklist

* [ ] Automated tests written and passing - we don't currently have browser specific testing
